src/platform/landing-pages/header-footer-data.json

```diff
@@ -196,22 +196,21 @@
     },
     {
       "column": 4,
-      "href": "https://www.va.gov/find-locations/",
+      "href": "https://www.va.gov/resources/",
       "order": 1,
       "target": null,
-      "title": "Find a VA location"
+      "title": "Resources and support"
     },
     {
       "column": 4,
-      "href": "https://iris.custhelp.va.gov/app/ask",
+      "href": "tel:18006982411",
```

Comment:
Y'all still waiting on the new /contact-us page to be deployed still?

Reply:
Ohhh that would make so much more sense lol I have no clue, but ya that's probably where they meant for this and the new header link to go

Reply:
Yupp you were right after confirming with Beth Potts, lmk when it's live if you could! 🙂

Reply:
@kelsonic I'm not exactly sure whom is publishing the contact-us page.
Gut feeling it is the CMS team if we already have a node template built for it?

Reply:
Ah gotcha, ya maybe! I think Beth will let us know 👍 thanks @zacharymorel!